tools/misc/tpm2_encodeobject.c

```diff
@@ -182,15 +182,12 @@ encode(void)
     }
 
     tpk->type = OBJ_txt2obj(OID_loadableKey, 1);
-    tpk->parent = ASN1_INTEGER_new();
-    tpk->privkey = ASN1_OCTET_STRING_new();
-    tpk->pubkey = ASN1_OCTET_STRING_new();
-    if (!tpk->type || !tpk->privkey || !tpk->pubkey || !tpk->parent) {
+    if (!tpk->type) {
         LOG_ERR("oom");
         goto error;
     }
 
```

Comment:
I thought we could delete this.

Reply:
Removing the OBJ_txt2obj call removes the OID_loadableKey from the generated PEM file. This causes problems for the openssl API when trying to parse the PEM file's BIO object into a tpk right before loading.